### PR TITLE
docs: improve api docs

### PR DESCRIPTION
**Pull request by fforootd**
- [x] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [x] PR is linked to the corresponding user story
- [x] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [x] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [x] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

**Comment by fforootd** [comment]
@hifabienne I configured redocusaurus to render the currently deployed json

Currently the files are grouped with the tags like these one https://github.com/zitadel/zitadel/blob/main/proto/zitadel/admin.proto#L731

I will integrate a generating step for the docs to ease development

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -221,6 +287,10 @@ module.exports = {
         },
       },
     ],
+    [
+      'redocusaurus',
+      process.env.NODE_ENV === "development" ? devConfig : prodConfig,
```
Comment (peintnermax) [inline docs/docusaurus.config.js:292]:
```suggestion
      process.env.NODE_ENV === "production" : prodConfig : devConfig,
```

Comment (fforootd) [reply docs/docusaurus.config.js:292]: haha, yes this works with me as well 😁

File: build/grpc/Dockerfile [hunk 2]
```diff
@@ -19,7 +19,7 @@ FROM ${BUILDARCH}-base
 ARG PROTOC_VERSION=3.18.0
 ARG PROTOC_ZIP=protoc-${PROTOC_VERSION}-linux-${PROTOC_ARCH}.zip
 ARG GRPC_WEB_VERSION=1.3.0
-ARG GATEWAY_VERSION=2.6.0
+ARG GATEWAY_VERSION=2.15.1
```
Comment (fforootd) [inline build/grpc/Dockerfile:22]: I needed to update the grpc-gateway to mitigate this error https://github.com/zitadel/zitadel/actions/runs/4247961435/jobs/7386579558#step:7:241
@livio-a @hifabienne  i hope this does not break anything

**Comment by fforootd** [comment]
Besides this typo here https://github.com/zitadel/zitadel/blob/redocusaurus/proto/zitadel/admin.proto#L2408 I cleaned out a lot of typos in the proto files. IMO its good to be released and improved up on.

File: proto/zitadel/admin.proto [hunk 3]
```diff
@@ -43,6 +44,84 @@ option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
...
+        {
+            name: "Authentication Methods"
+        },
+        {
+            name: "Branding",
+            description: "Defines the look of the login UI, E-Mails and ZITADEL Console. For adding assets like logo, icon and font, have a look at the assets API."
+        },
+        {
+            name: "Deprecated"
+        },
```
[the remark this answers is not in the document]
Comment (hifabienne) [reply proto/zitadel/admin.proto:56]: Depends where we want to list them. I changed it to the place with what they were replaced.

image

**Review by hifabienne — APPROVED** [review]

**Pull request merged**